treatment_code_quantities.csv

```diff
@@ -1,48 +1,34 @@
-"Treatment Code","Quantity","Address Level"
-HH_1LSFN,11756,U
-HH_2LEFN,5170,U
-HH_3QSFN,2369,U
-HH_LF2R1E,6723,U
-HH_LF2R1W,104,U
-HH_LF2R2E,5946,U
-HH_LF2R2W,427,U
-HH_LF2R3AE,6157,U
-HH_LF2R3AW,642,U
-HH_LF2R3BE,386,U
-HH_LF2R3BW,214,U
-HH_LF3R1E,16191,U
-HH_LF3R1W,761,U
-HH_LF3R2E,18180,U
-HH_LF3R2W,1264,U
-HH_LF3R3AE,21108,U
-HH_LF3R3AW,1196,U
-HH_LF3R3BE,471,U
-HH_LF3R3BW,556,U
-HH_LFNR1E,62374,U
-HH_LFNR1W,2345,U
-HH_LFNR2E,63872,U
-HH_LFNR2W,4454,U
-HH_LFNR3AE,76888,U
-HH_LFNR3AW,4646,U
-HH_LFNR3BE,1038,U
-HH_LFNR3BW,2128,U
-HH_QF2R1E,463,U
-HH_QF2R1W,222,U
-HH_QF2R2E,573,U
-HH_QF2R2W,237,U
-HH_QF2R3AE,274,U
-HH_QF2R3AW,594,U
-HH_QF3R1E,1912,U
-HH_QF3R1W,1737,U
-HH_QF3R2E,2882,U
-HH_QF3R2W,1004,U
-HH_QF3R3AE,870,U
-HH_QF3R3AW,805,U
-HH_QFNR1E,3566,U
-HH_QFNR1W,5060,U
-HH_QFNR2E,4859,U
-HH_QFNR2W,3233,U
-HH_QFNR3AE,1426,U
-HH_QFNR3AW,3571,U
-SPG_QDHSE,747,U
-SPG_QDHSW,836,U
\ No newline at end of file
+Treatment Code,Quantity,Address Level
+HH_LP1E,10606,U
```

Comment:
Why 10,606? Adds up to 350k I suppose?

Reply:
Yes. Vicki doesnt have the actual quantities yet so just gave them all the same amount for now